configuration/security.md

```diff
@@ -81,13 +81,13 @@ Placing `sudo` in front of a command runs it as a superuser, and by default, tha
 To force `sudo` to require a password, enter:
 
 ```bash
-sudo nano /etc/sudoers.d/010_pi-nopasswd
+sudo visudo /etc/sudoers.d/010_pi-nopasswd
```

Comment:
Not related to this change, but I just noticed that the previous paragraph on this page mentions "require superuser credential" which should probably be "require superuser credentials" ?

Reply:
Good spot - fixed.